nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -359,17 +384,23 @@ def _get_quantization_target_points(self, model: TModel) -> OrderedDict[TargetPo
         pattern = PatternsManager.get_full_pattern_graph(backend, device)
         quantizer_setup = self._get_quantizer_setup(nncf_graph, pattern)
         self._apply_model_type_pass(self._parameters.model_type, quantizer_setup, nncf_graph)
-        for quantization_point in quantizer_setup.quantization_points.values():
+        quantization_points = list(quantizer_setup.quantization_points.values())
+        quantization_points = self._sort_quantization_points(quantization_points, nncf_graph)
```

Comment:
Why do we need quantization points sorting?

Reply:
As `QPS` doesn't return the `quanter_setup` in the topological order, while we seek a way to detect the first weight quantizer for possibly apply `overflow_fix` to it.

Reply:
But, are you confident in topological sorting? I mean, it would be better to determine the first layers for the correction more "algorithmically".

Reply:
In such scenario, we lost in performance, but the logic becomes more robust.
I would keep sorting anyway and add a way to determine the quantizers for applying overflow_fix.
What do you think?

Reply:
In comparison to the statistics collection step, the performance losses are not dramatic, I think.
I wonder only about the robustness of the process. Also, what would happen if one of the first layers with the weight will be in the ignored scope? Which layer will be determined as the first?
cc @alexsu52

Reply:
Added explicit search of quantizers to apply overflow fix